clients/runtime/Cargo.toml

```diff
@@ -31,7 +31,7 @@ log = "0.4.0"
 serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.71"
 thiserror = "1.0"
-tokio = { version = "1.0", features = ["full"] }
+tokio = { version = "1.37", features = ["full"] }
```

Comment:
The reason for this is the "lost its waker" issue in tokio, [fixed in the newer versions](https://github.com/tokio-rs/console/issues/345#issuecomment-1817939351).